src/mlpack/methods/ann/loss_functions/mean_squared_error.hpp

```diff
@@ -42,24 +42,26 @@ class MeanSquaredError
   /**
    * Computes the mean squared error function.
    *
-   * @param input Input data used for evaluating the specified function.
+   * @param prediction Predictions used for evaluating the specified loss
+   * function.
```

Comment:
This should be tabbed twice (4 spaces), see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping for more details.